test/button-state.test.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
```

Comment:
My aim was for all specs to be moved to WCT, including pure javascript, React, and Angular specs. That way, we'll only have two test suites:

1. WCT for all javascript/component testing
2. Gemini + Storybook for visual regression tests

Once WCT launches, it's no slower than any other test runner at running pure javascript specs. Having a third test runner for pure javascript tests would slow tests and add overhead.

The React spec still lives in the old test runner/suite. I planned to move that over as a part of my next story. But I can move them over as a part of this PR if you'd like.